src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -1,47 +1,44 @@
+### Configure the client to point to the AWS AppSync API endpoint
 
 You can use existing AWS AppSync resources by referencing the AWS AppSync API endpoint and configuring the parameters in your app. The API URL, ID, and Key can be found in the AWS AppSync console in the Settings section. You will need to know your Authentication type to complete this step. The options for Authentication type are `API_KEY`, `AWS_IAM`, `AMAZON_COGNITO_USER_POOLS`, and `OPENID_CONNECT`.
 
-#### Review differences between authentication types 
-
-[comment]: # (EXPANDER SECTION to provide a deeper dive on authentication types and their differences, including API key access, Cognito, IAM, and OIDC. May be an option for a table that compares the authentication types. Content copied and updated from https://github.com/aws-amplify/docs/blob/main/src/pages/cli/graphql/authorization-rules.mdx However, I would suggest we rewrite this intro sentence to provide more foundational explanation that we can then reuse.)
+<Accordion title='Review differences between authentication types' headingLevel='4' eyebrow='Learn more'>
 
 The following table compares the different authentication types and their use cases. 
 
 | **Recommended use case** | **Strategy** | **Provider** |
 |---|---|---|
-| Public data access where users or devices are anonymous. Anyone with the AppSync API key is granted access. | [`public`]() | `apiKey` |
+| Public data access where users or devices are anonymous. Anyone with the AWS AppSync API key is granted access. | [`public`]() | `apiKey` |
 | Recommended for public data access in a production environment, or where unauthenticated users or devices are granted permissions using AWS IAM controls. | [`public`]() | `iam` |
 | Per-user data access. Access is restricted to the "owner" of a record. Leverages `amplify add auth` Cognito user pool by default. | [`owner`]() | `userPools` / `oidc` |
 | Any signed-in data access. Unlike owner-based access, **any** signed-in user has access. | [`private`]() | `userPools` / `oidc` / `iam` |
 | Per-user group data access. A specific or dynamically configured group of users have access. | [`group`]() | `userPools` / `oidc` |
 | Define your own custom authorization rule within a Lambda function. | [`custom`]() | `function` |
 
-[comment]: # (EXPANDER end)
+</Accordion>
 
-Once you have these details, you can configure the client using the following options:
+<br/>
 
-[comment]: # (Content copied and updated from https://github.com/aws-amplify/docs/edit/main/src/fragments/lib/graphqlapi/js/existing-resources.mdx )
+Choose your authentication type from the tabs below and configure the client by updating the following details:
 
 <BlockSwitcher>
 <Block name="API_KEY">
-[question]: # (RENE, We sometimes use ```javascript and other times use ```js at the beginning of code blocks. Do we need to be consistent? Same question applies to the TypeScript blocks.)
+
```

Comment:
Q from Jim during his initial edit pass - We sometimes use javascript and other times use js at the beginning of code blocks. Do we need to be consistent? Same question applies to the TypeScript blocks.

Reply:
no need to be consistent. It's whatever's convenient. In the end the tool will automatically map js -> javascript and ts -> typescript.